src/shared/components/query/QueryStore.ts

```diff
@@ -1,1734 +1,1740 @@
-/* tslint:disable: indent linebreak-style */
-import * as _ from 'lodash';

```

Comment:
Woah. I'm going to go check why this diff is so insane.

Reply:
Ah, line endings

Reply:
Ok, I don't know what to do about this. I changed my `core.autocrlf` settings to `false` both globally and locally and this file is still getting nuked, presumably by line endings.